packages/x-date-pickers/src/internals/components/PickersToolbar.tsx

```diff
@@ -51,7 +51,7 @@ const PickersToolbarRoot = styled('div', {
   justifyContent: 'space-between',
   padding: theme.spacing(2, 3),
   ...(ownerState.isLandscape && {
-    height: 'auto',
+    height: '100%',
```

Comment:
That modification was a mistake. I thought it would fix Argos diff. But those diffs were due to a modification of the layout: Now the action bar goes below the toolbar in landscape mode. Before the toolbar was covering the entire height

![image](https://user-images.githubusercontent.com/45398769/206158018-dac16431-91d0-4704-984e-e11b91aa4dec.png)

I reverted this `height` to `auto`, and I modified the behavior such that in landscape mode, it's a `flex-start` instead of a `space-between` which sounds more natural

![image](https://user-images.githubusercontent.com/45398769/206157935-c240c1ee-12f9-480a-87a3-a3d0a310ab79.png)